src/sinks/aws_s3/config.rs

```diff
@@ -35,29 +35,83 @@ const DEFAULT_KEY_PREFIX: &str = "date=%F/";
 const DEFAULT_FILENAME_TIME_FORMAT: &str = "%s";
 const DEFAULT_FILENAME_APPEND_UUID: bool = true;
 
-#[derive(Deserialize, Serialize, Debug, Clone)]
-#[serde(deny_unknown_fields)]
+/// Configuration for the `aws_s3` sink.
+#[configurable_component(sink)]
+#[derive(Clone, Debug)]
 pub struct S3SinkConfig {
+    /// The S3 bucket name.
+    ///
+    /// This must not include a leading `s3://` or a trailing `/`.
```

Comment:
A thought on consistency, which you'd mentioned offline, I remember seeing both `must` and `_must_`. Is this something we'd like to add to any "writing guidelines" we end up having?

Reply:
Hmm, probably. Haven't even really gotten into worrying too much about the styling of the doc comments.

Reply:
Yeah, wasn't a major thing - was just top of mind based on your parting thoughts last night. Most of the comments do appear to be using `_must_`, if memory serves.